boards/arm/96b_wistrio/96b_wistrio.dts

```diff
@@ -86,7 +85,7 @@
 	status = "okay";
 	cs-gpios = <&gpiob 0 GPIO_ACTIVE_LOW>;
 
-	lora: sx1276@0 {
+	lora0: lora@0 {
```

Comment:
This commit does two things:
- change node name from something like `sx1276` / `sx1262` / `radio` to `lora`,
- drop `/aliases/lora0` in favour of `lora0` nodelabel.

The first change in justified in commit message. However the second change does't seem to be related to the first and I also do not see the reason why we should use nodelabel instead of an alias. I'd like second commit message for the second change, with a good reason in commit body why we want that.

Reply:
Hmm, since those two changes were interdependent I thought having them together will help in bisecting. But I don't have a strong opinion either, so will split it up.

Reply:
It should be:
```
lora: lora@0 {
```
in this commit, so that `/aliases/lora0` is still valid.